app/workers/scheduled_worker_runner.rb

```diff
@@ -33,9 +33,17 @@ def self.write_worker_history(worker_string, record, value = Time.current)
   end
 
   def self.scheduled_workers
-    [UpdateExpiredInvoiceWorker, UpdateCountsWorker, UpdateOrganizationPosKindWorker,
-     FileCacheMaintenanceWorker, UnusedOwnershipRemovalWorker, TsvCreatorWorker,
-     GetManufacturerLogoWorker, DeactivateExpiredTheftAlertWorker] + [self]
+    [
+      UpdateExpiredInvoiceWorker,
+      UpdateCountsWorker,
+      UpdateOrganizationPosKindWorker,
+      FileCacheMaintenanceWorker,
+      UnusedOwnershipRemovalWorker,
+      TsvCreatorWorker,
+      GetManufacturerLogoWorker,
+      DeactivateExpiredTheftAlertWorker,
+      EmailHeldBikeNotificationWorker,
+    ] + [self]
```

Comment:
Nit: can we just include `self` in the regular array? (I know I did this, just seeing it now and wondering)

Reply:
I left it as-is out of an abundance of caution, I'm not sure what the intent was but I'll update 👍

Reply:
Sweet. I have no idea why I left it out originally 🤷‍♂️ - I assume it was just foolishness on my part.